### PR TITLE
Option to skip RequestBody validation

### PR DESCRIPTION
**Pull request by merk**
I have a requirement that the RequestBodyParamConverter doesnt perform the validation, so my controller action can calculate which validation groups are required and do it itself.

Seems to me like its logical that this is configurable on ParamConverter.

### CODE REVIEW

File: Request/RequestBodyParamConverter.php [hunk 1]
```diff
@@ -105,7 +105,7 @@ public function apply(Request $request, ParamConverter $configuration)
 
         $request->attributes->set($configuration->getName(), $object);
 
-        if (null !== $this->validator) {
+        if (null !== $this->validator && (!isset($options['skipValidation']) || !$options['skipValidation'])) {
```
Comment (Ener-Getick) [inline Request/RequestBodyParamConverter.php]: What about `validate`?

Comment (Ener-Getick) [inline Request/RequestBodyParamConverter.php]: It looks very verbose, let's keep something short :)

Comment (merk) [inline Request/RequestBodyParamConverter.php]: Changed to `validate`

**Comment by merk** [comment]
@GuilhemN is there anything stopping this from being merged?

**Comment by GuilhemN** [comment]
Thank you @merk!

**Pull request merged**